ani-cli

```diff
@@ -138,7 +138,12 @@ dep_ch () {
 download () {
 	case $2 in
 		*m3u8*)
-			ffmpeg -loglevel error -stats -referer "$1" -i "$2" -c copy "$download_dir/$3.mp4" ;;
+			if [ ! -d "$download_dir" ] ; then
+				die "Directory \"$download_dir\" does not exist."
+			else 
+				ffmpeg -loglevel error -stats -referer "$1" -i "$2" -c copy "$download_dir/$3.mp4"
```

Comment:
The else block would become obsolete with the suggested change above.
ffmpeg would then need to run outside the if-else.